src/after.js

```diff
@@ -8,7 +8,7 @@ function After$race(other){
        ? other
        : isNever(other)
        ? this
-       : (other instanceof After || other instanceof RejectAfter)
+       : typeof other._time === 'number'
```

Comment:
Technically, no. It's used in some places (like `isFuture`) to determine with high performance whether a value is what we want it to be, but these are the only occurrences left where there are no *or* conditions for the value to be identified as such.